ocaml/xapi/xapi_host.ml

```diff
@@ -1171,6 +1171,24 @@ let install_server_certificate ~__context ~host ~certificate ~private_key ~certi
   (* sever all connections done with stunnel *)
   Xapi_mgmt_iface.reconfigure_stunnel ~__context
 
+let emergency_server_certificate_reset ~__context =
+  let generate_ssl_cert = "/opt/xensource/libexec/generate_ssl_cert" in
```

Comment:
This of course need to get defined in xapi_globs.ml, but I think that @lippirk has done that already on his feature branch.

Reply:
The intention is to remove the bash script altogether. Not sure if the merge between the branches will come before or after I get around to replace it.